src/category_theory/limits/preserves/basic.lean

```diff
@@ -278,6 +292,20 @@ def preserves_colimits_of_shape_of_equiv {J' : Type w₂} [category.{w₂'} J']
       { dsimp, simp [←functor.map_comp] }, -- See library note [dsimp, simp].
     end } }
 
+/--
+`preserves_colimits_of_size_shrink.{w w'} F` tries to obtain `preserves_colimits_of_size.{w w'} F`
+from some other `preserves_colimits_of_size F`.
+-/
+def preserves_colimits_of_size_shrink (F : C ⥤ D)
+  [preserves_colimits_of_size.{(max w w₂) (max w' w₂')} F] : preserves_colimits_of_size.{w w'} F :=
+⟨λ J hJ, by exactI preserves_colimits_of_shape_of_equiv
+  (ulift_hom_ulift_category.equiv.{w₂ w₂'} J).symm F⟩
+
+/-- Preserving colimits at any universe implies preserving colimits at universe `0`. -/
+def preserves_smallest_colimits_of_preserves_colimits
+  (F : C ⥤ D) [preserves_colimits_of_size.{v₃ u₃} F] : preserves_colimits_of_size.{0 0} F :=
+preserves_colimits_of_size_shrink F
```

Comment:
Out of curiosity, is this lemma useful to you? The `has_smallest_limits_of_has_limits` was added in the first place since I thought it could be made into an instance but I was obviously wrong. I did't bother to remove it though.

Reply:
That's what I suspected, but maybe it's good to have it in because it'll be the most common use case of `preserves_colimits_of_size_shrink` and is usable without stating universe parameters?